tests/unit/swagger/schemaDetails3.spec.ts

```diff
@@ -1058,48 +1054,4 @@ describe('Definition generation for OpenAPI 3.0.0', () => {
       });
     });
   });
-
-  describe('illegal input values', () => {
```

Comment:
Nice work @WoH. Can we replace this deleted test with a positive test so people can see how the enum of strings and numbers becomes a union of two schemas?

Reply:
Sure, good point!

Reply:
Added: a9e7b50